README.md

```diff
@@ -1,4 +1,13 @@
-## Web Console
+## OpenShift 4.0 Designs
+Access documentation for specific sections by clicking on the links below
+- [Displaying Status](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/status)
+- [Code Editors](http://openshift.github.io/openshift-origin-design/web-cosnsole/4.0-designs/code-editors)
```

Comment:
typo in URLs for [Code Editors], [Actions], and [Labels]: **web-console**

Reply:
Thanks @tlwu2013, just made that update!